deploy/olm-catalog/portworx/99.9.9/portworx-certified.clusterserviceversion.yaml

```diff
@@ -71,11 +71,11 @@ spec:
+PMXWGc0QGDyMER69IXV4WoEli5A4LNliAwcf0TeRxo5+gj4lriUCZhNMUrr02CQWahmEeluFJExJ6lwB2eNoqYoR8dy/rKGJnJ0EDDoF5XsjDQEX3mBtJqKMNvR7rTbNiHw63fu3uvG+h8VOZmNLorphF2/ISL30ioGEv55r6T/zkOky/UwfvseVuvOKj/w0VuwGrUG9u5CkF7Yvgk4rgbMeo1grFtNv1K+Gjb9DFgMcW7egMAXy5E1j7bzoR+tMP78HcF3/w9W2gFFWoFNfyD48UKnf/Tr5sEltJqC4t/Mi7uR+h5W7aJcBWpjJaxWl8GufhICq96n+9oMs1EboEZt1TekHqC6b5G2WBtWJSd41a5Fi7gzMxBcOB8G4WK2vxo2LeQXKToC/7BopoLfSPLmTJT+fj+SUyMIk+M9mUgrzC8651dS9KXPw6SesfnI5fiVROciRw+B4LxneWtVGJ98hNBzQ2GXKoPg80OVFsXvGpw3EbY2C/PqRjky5TLTEfjrfwgschaGc9HQ60SIu7cj8NM6BMfeD2P9T7BoF9fQ2y8hNPsZ0tjOQHDOBEUUumlFFCVLI0T9Mojs7JqnInIn9YfIyT6+Jmzatoh/R8DYvRPhvp1hZJMGv3MLwgMup188JyJbsRBJI/uTH5XyKUA49NRABJYvhF2xCsLDbofx89ek8achfM8VYPICkXTS6LsQXPkeuS5MhB+4lszgrUC5ikgaSOcpu3XX5FgEBOgj7E+xh74B66Wx9G251HMDZAqyaUic5JqEvI0NcxQdlANfmYhULkgZIkcNAbNrT5iXdgc69ED45ouA3oNhnd0YwTVLYfKC9h8+pyVXow5+/43/Q/DtmaRF70Pgg9dh3kfaF/0KeV5inXke/V7hEHU5MGoAssncs0+qBbNCFQRmPQ2zI5EEiUFaVvZTryDw9iwk9SMyot8LMG8eCPM8WvBdtz5sIkerWTuE3pgO68JWiHS72alHmp4iH0qZ7a+C2eMOlR/pTYu6S5WFQeRrEfGxNgnSKpk0zWvuVGUC639Qx+CXK2GHk2Fe7awlDXGbqxfBJDNVpGgI+Ja4UP1E8kk8DeO7tQhOf5zMBXK4RglLQcFkRWLH8BM76x0WEx+Xg89R+3vciU7TJUoCZcrBSNsPu3VHGGSOBUgTs+s3AcqWP+jtDdJOQD8uYVWlxe/Dp8KufSaCny45qJyb4W2DgvaC788njchZO2mdc6FbjE/YvDOJ5EwineDapQiOewDm7I9jyoB+xNWufHxOXpUTSFOiQEASo3QZNz9EpMiOfqvehbSsLEL59FzR+9tVj3PL2JWqOed7dhIR70Vo5jj3GkhTEyk6Ar41FdUt86xhvcaIjHkdVq9B9GCVpG+2AKwkCnkI0zU+Z0d9UsB5URrJZDPSi8uIHEUEvv3MGaKtG8ncI58X+XTMJm3JvPsKgY9IW6afo89NLPZVdbpeOc2ZtJSQthLY9hdpXpkwSDNiMzE3sU+uC7tNN0TuGgXzytuAsy9wixmkEbEJqMw4Mg8t0sjAPxxMX2S00pXUKSYfSlIbBmmDSohEAz99CdQ6zUnrvxlkvpJDP3vUTNLaHqBnK1ldsWuciiD52Nifxa/Ad5+qfJPy1aL+fsNV32zyiVk16+jW5FgEBPyrcXlvlh5sq921sBteArw5EcaahXTV+SVqg/bbiipfHm1LPa/eFuT8CCJg0xeKsWcXkgbfBGP7ZjLlBitfEvuTrPObI7DuE2Rd2LrA72ie0xCB0hUQvuUSWHXOIk3NswbR8/2TTT6r8JP3wDjhZPKLrUek1/3ue9inngXrnEYI33EZmYnkGCeHfqT3g87ea3XrITD9CYTI1IwMfAqhTxcjfPeV5JvKhnV6fZiN2yBEGporpEVa59KuGINugE1bIRlpqRyBoyYH+Mdfw7e2Ve9hVavhVKHQC/P8Fgj37wabJiuM1BSYI2a6zclJ4RHwbQBqfrdq/7YOgZfHwN70qxv2oGYdlX/LqalMxsnRb9b8GpNrhUbAJm3DoJk4td6KtRFyzGsJvTCSCMFyHOM6M3q0SZMxeKkDmZYHCWlEBplcysnO67jKlIdNO0AoM6102Zzi5CA3Nm+gH76sRiSR81NnuoByzKfsA6pWp/3XSutsR5uj8Att3hk7NpNtSNp6lajZmEpbhRPxgk1fFu4nkaNBJp/NMWd8jSYIDNIomSRpjxuEJgwByDyNXEu/RkXCbdrczkm1HcJUufKnKAgUS+JSQNCDbq9ZAHvBczTbk+o8hGQWKN8WHVkLC05YWxTMpE5RECDMlQZGDuzsJ+bCpg0ci51QaEV4cE9Y7HcjwmaHffao2bBrnFLsbvXvvqHiS1wa2f27YH0wDfZaisthwqKXEjIxQmPX6FJyPNoIEO48A2dTOADtEHi03+3va599ZTRrqGLZOH4rN+3x7+tdsXnn4k9c0aGy//oB1jtP07S3M0XN2aGRq4rNQMqNCAKJhIC/ZxULMVLGSWcheMcLCFwxGHaFymq2sRDVpaggIAj8gxBIGI0rBvOMA4ismInQpRQJLSIICAK+QyAxict3wyQdFgQEAS8CCWMqem9azgUBQcDfCAhx+Xv8pPeCQEIiIMSVkMMuNy0I+BsBIS5/j5/0XhBISASEuBJy2OWmBQF/IyDE5e/xk94LAgmJgBBXQg673LQg4G8EhLj8PX7Se0EgIREQ4krIYZebFgT8jYAQl7/HT3ovCCQkAkJcCTnsctOCgL8REOLy9/hJ7wWBhERAiCshh11uWhDwNwJCXP4eP+m9IJCQCAhxJeSwy00LAv5GQIjL3+MnvRcEEhIBIa6EHHa5aUHA3wgIcfl7/KT3gkBCIiDElZDDLjctCPgbASEuf4+f9F4QSEgEhLgSctjlpgUBfyMgxOXv8ZPeCwIJiYAQV0IOu9y0IOBvBIS4/D1+0ntBICEREOJKyGGXmxYE/I2AEJe/x096LwgkJAL/D1ntAHk5wINSAAAAAElFTkSuQmCC
```

Comment:
Let's add a better quality image with transparent background.

Reply:
Added better quality transparent image from https://portworx.com/brand/
added screenshot in comments of the PR.

Reply:
Wow.. I can't believe Piyush actually decoded and reviewed the image as a part of this review  (that's spells "dedication"  ;) )

Reply:
Haha.
@nikita-bhatia this is a much better image. It is a 3456×1323 image currently. Can you reduce it to something smaller like 320 or 640 as the height of the image?

Reply:
@piyush-nimbalkar updated image pixel size to 640*245, please check.@piyush-nimbalkar updated image pixel size to 640*245, please check.